CoreScriptsRoot/CoreScripts/PurchasePromptScript2orig.lua

```diff
@@ -0,0 +1,1531 @@
+--[[
```

Comment:
Hmm, apparently an autocomplete error.

On Thu, Jun 2, 2016 at 10:44 AM, Ben Tkacheff <notifications@github.com>
wrote:

> In CoreScriptsRoot/CoreScripts/PurchasePromptScript2orig.lua
> <https://github.com/ROBLOX/Core-Scripts/pull/397#discussion_r65585508>:
>
> > @@ -0,0 +1,1531 @@
> > +--[[
>
> I don't think we want to add this file
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ROBLOX/Core-Scripts/pull/397/files/8ceb906c0315af7147392bd9dfb56cb119ada5d6#r65585508>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AO6ssqs83NZW4v_Or1itHIFzaQ9b2Blnks5qHxaQgaJpZM4IitY7>
> .
>